tests/functional/recovery_key.js

```diff
@@ -106,6 +117,10 @@ registerSuite('Recovery key', {
         .then(openVerificationLinkInSameTab(email, 2))
         .then(testElementExists(selectors.COMPLETE_RESET_PASSWORD_RECOVERY_KEY.HEADER))
 
+        .then(fillOutRecoveryKey('N8TVALID'))
+        .then(visibleByQSA(selectors.COMPLETE_RESET_PASSWORD_RECOVERY_KEY.TOOLTIP))
```

Comment:
Like `click`, `testElementTextIncludes` already has a `visibleByQSA` so this is unnecessary.